src/plugins/embeddable/public/index.ts

```diff
@@ -85,6 +86,7 @@ export type {
   SavedObjectEmbeddableInput,
   SelfStyledEmbeddable,
   ValueClickContext,
+  HasEditorApp,
```

Comment:
If an embeddable has a custom editor app, they will add this to their API - so, it needs to be exported (even though it's not used yet)